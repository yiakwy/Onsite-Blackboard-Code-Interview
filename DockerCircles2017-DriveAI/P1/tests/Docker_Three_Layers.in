7
Ubuntu10.1,Master Ubuntu10.2,Master Ubuntu10.3,Master vm1,Ubuntu10.9 vm2,Ubuntu10.2 division1,Ubuntu10.2 Ubuntu11.2,google-server
