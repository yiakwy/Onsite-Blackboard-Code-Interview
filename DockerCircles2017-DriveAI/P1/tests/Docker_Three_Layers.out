Input the Number of imgs dependencies: ------

─── Node: <Master> 
    ├── Node: <Ubuntu10.1> 
    ├── Node: <Ubuntu10.2> 
    |   ├── Node: <vm2> 
    |   └── Node: <division1> 
    └── Node: <Ubuntu10.3> 
------

─── Node: <Ubuntu10.9> 
    └── Node: <vm1> 
------

─── Node: <google-server> 
    └── Node: <Ubuntu11.2> 
Master Ubuntu10.1 Ubuntu10.2 Ubuntu10.3 vm2 division1 Ubuntu10.9 vm1 google-server Ubuntu11.2 
